### ISSUE TITLE
vmselect returns no data for too big look-behind window like `foo[100y]`

### ISSUE THREAD

**Issue by dandv** [VictoriaMetrics/VictoriaMetrics#5553]
### Describe the bug

If you want to cover the *entire* time range in a series, it seems that adding a `[100y]` duration should work. But actually it doesn't, because the start of that interval would be before 1970-01-01.

### To Reproduce

```sh
curl http://localhost:8428/api/v1/import/prometheus -d 'oldtimes 123'
curl http://localhost:8428/api/v1/query -d 'query=oldtimes[10y]'  # shows the point
curl http://localhost:8428/api/v1/query -d 'query=oldtimes[100y]'  # nothing
```

### Version

victoria-metrics-20231212-224437-tags-v1.96.0-0-g304fe0565

### Logs

Nothing

### Used command-line flags

-retentionPeriod=100y -search.maxPointsPerTimeseries=10000000

**Comment by hagen1778** [comment]
@dandv the issue was addressed in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5630. The fix will be available in the next release.

**Comment by valyala** [comment]
VictoriaMetrics should properly process queries with too big look-behind windows in queries starting from [v1.97.0](https://docs.victoriametrics.com/CHANGELOG.html). Closing the issue as fixed.

**Comment by valyala** [comment]
FYI, the bugfix for this issue has been also included in [v1.87.14 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.14)

**Comment by valyala** [comment]
FYI, the bugfix for this issue has been also included in [v1.93.11 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.11).
